lib/aerospike/command/command.rb

```diff
@@ -429,17 +429,20 @@ def execute
 
       # set timeout outside the loop
       limit = Time.now + @policy.timeout
```

Comment:
Do you think `Time.now` is enough here and in line 446? Perhaps we should use `Process.clock_gettime(Process::CLOCK_MONOTONIC)`?